### DISCUSSION TITLE
Meu primeiro package público: Small-Cache, a maneira mais minimalista e direta de cachear suas requests no frontend

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#2150] [Mostre seu trabalho]
Faaala pessoal, espero que todos estejam bem!

Venho apresentar a vocês, minha primeira lib publicada node:
🎈 A maneira mais simples, fácil e minimalista 😃 de fazer cache de suas solicitações em seu front-end (JS / TS, com React.js, Vue.js, Angular e todos os outros)

https://www.npmjs.com/package/small-cache

Espero que ajude, e estou aberto a sugestões =)

**Comment by HigorAln** [comment]
God!
